src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -69,7 +69,9 @@ def retrieve_electrical_series(nwbfile: NWBFile, electrical_series_name: Optiona
 
 
 def read_nwbfile(
-    file_path: str | Path,
+    *,
+    file_path: str | Path | None,
+    file,
```

Comment:
from typing import IO?
Is that what file-like means?

Reply:
I have added the annotations using BinaryIO

Reply:
Thanks!